velox/connectors/fuzzer/FuzzerConnector.h

```diff
@@ -56,7 +56,8 @@ class FuzzerDataSource : public DataSource {
   FuzzerDataSource(
       const std::shared_ptr<const RowType>& outputType,
       const std::shared_ptr<connector::ConnectorTableHandle>& tableHandle,
-      velox::memory::MemoryPool* FOLLY_NONNULL pool);
+      velox::memory::MemoryPool* FOLLY_NONNULL pool,
+      const std::shared_ptr<dwio::common::ReaderOptions>& options = {});
```

Comment:
FuzzerDataSource extends DataSource. I don't see why we have to add this argument. DataSource API did not change.